code/modules/mob/living/carbon/alien/humanoid/caste/hunter.dm

```diff
@@ -18,11 +18,14 @@
 //Hunter verbs
 
 /mob/living/carbon/alien/humanoid/hunter/proc/toggle_leap(message = 1)
+	if(pounce_cooldown)
+		src << "<span class='alertalien'>You are too fatigued to pounce right now!</span>"
+		return
 	leap_on_click = !leap_on_click
 	leap_icon.icon_state = "leap_[leap_on_click ? "on":"off"]"
 	update_icons()
 	if(message)
-		src << "<span class='noticealien'>You will now [leap_on_click ? "leap at":"slash at"] enemies!</span>"
+		src << "<span class='noticealien'>You will now [leap_on_click ? "metabolize plasma to leap at":"slash at"] enemies!</span>"

```

Comment:
It really doesn't. Lore-wise just jumping wouldn't cost anything. Remove this. The fatigue cool down is enough and makes sense already. If you want to nerd jump, make the time on that slightly higher.